packages/jellyfish-api-core/__tests__/category/loan/loanPayback.test.ts

```diff
@@ -0,0 +1,459 @@
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import { GenesisKeys } from '@defichain/testcontainers'
+import BigNumber from 'bignumber.js'
+import { TestingGroup } from '@defichain/jellyfish-testing'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+const tGroup = TestingGroup.create(2, i => new LoanMasterNodeRegTestContainer(GenesisKeys[i]))
+const alice = tGroup.get(0)
+const bob = tGroup.get(1)
+let bobVaultId: string
+let bobVaultAddr: string
+let bobVaultId1: string
+let bobVaultAddr1: string
+let bobLiqVaultId: string
+let bobloanAddr: string
+let bobColAddr: string
+
+async function setup (): Promise<void> {
+  // token setup
+  const aliceColAddr = await alice.container.getNewAddress()
```

Comment:
You can keep   

let bobColAddr: string
let bobVaultAddr1: string

inside the setup function